### PR TITLE
website/docs: add new content from old PR #9524

### PR DESCRIPTION
**Pull request by tanberry**
This new content with steps about how to add a custom LDAP property mapping is thanks to @ChenWenBrian who originally opened PR #9524, but due to me not looking at it earlier and it getting completely stale, past the point of manageable merge conflicts, I have recreated his PSR here. Thanks Brian!!

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by rissson** [comment]
Let's add a `Co-authored-by: "Brian, Chen" <wildfox@126.com>` when merging.

**Review by dominic-r — CHANGES_REQUESTED** [review]

**Comment by tanberry** [comment]
@marc can you take a look at the `lint(black)` and `lint(ruff)` check fails? I'm not sure why a .py file is being discussed, lol?

If it's a check we can safely ignore, please approve PR and I'll get it merged. Thanks!

**Review by rissson — APPROVED** [review]

File: website/docs/sources/ldap/index.md [hunk 1]
```diff
@@ -73,6 +73,22 @@ By default, authentik ships with [pre-configured mappings](../../property-mappin
 
 You can assign the value of a mapping to any user attribute, or save it as a custom attribute by prefixing the object field with `attribute.` Keep in mind though, data types from the LDAP server will be carried over. This means that with some implementations, where fields are stored as array in LDAP, they will be saved as array in authentik. To prevent this, use the built-in `list_flatten` function.
 
+### Custom LDAP Property Mapping
+
+If the default source mapping is not enough, you can set your own custom LDAP property mapping.
+
+For example, the setting `ldap-displayName-mapping:name` means that the LDAP source field `displayName` will be mapped to the `name` field in authentik.
```
Comment (BeryJu) [inline website/docs/sources/ldap/index.md:80]:
```suggestion
```

Comment (rissson) [reply website/docs/sources/ldap/index.md:80]: ah

**Pull request merged**